doc/faq.rst

```diff
@@ -0,0 +1,89 @@
+==========================
+Frequently Asked Questions
+==========================
+
+Below are some of PySB's frequently asked questions. If your question is
+not answered here, you can try our
+`Gitter channel <https://gitter.im/pysb/pysb>`_.
+For more general Python related questions, we recommend `Stack
+Overflow <https://www.stackoverflow.com>`_.
+
+General
+=======
+
+* Do you support Python 3?
+
+    **Yes**. The current release of PySB supports Python 3.6 and
+    Python 2.7. Other Python 3.x releases but are not explicitly tested.
```

Comment:
I think there's an extraneous "but" in this sentence.

Reply:
Fixed.